.vscode/tasks.json

```diff
@@ -96,150 +84,154 @@
+                "command": "if [ ! -f '/etc/udev/rules.d/${config:linux_udev_ledgerblue_rule_file}' ]; then sudo cp .vscode/${config:linux_udev_ledgerblue_rule_file} /etc/udev/rules.d/ && sudo udevadm control --reload-rules && sudo udevadm trigger; fi",            
+            }, 
+            "osx" : {
+                // Checks that virtual env is installed, otherwise installs it. Then installs ledgerblue in a virtualenv.
+                "command": "[ -n '$VIRTUAL_ENV' ] || if ! python3 -m virtualenv --version >/dev/null 2>&1; then python3 -m pip install virtualenv; fi && [ -d 'ledger' ] || python3 -m virtualenv ledger && source ledger/bin/activate && python3 -m pip show ledgerblue >/dev/null 2>&1 || python3 -m pip install ledgerblue",            
+            }, 
+            "windows" : {
+                // Checks that virtual env is installed, otherwise installs it. Then installs ledgerblue in a virtualenv.
+                "command": "cmd.exe /C 'if not exist ledger (python -m pip install virtualenv && python -m venv ledger && call ledger\\Scripts\\activate.bat && python -m pip install ledgerblue)'",            
+            },
             "group": {
                 "kind": "test",
                 "isDefault": true
             },
             "problemMatcher": []
-        },
+        },        
         {
-            "label": "Install app loading requirements (macOS)",
+            "label": "[setup] Install vscode requirements",
             "type": "shell",
-            // Checks that virtual env is installed, otherwise installs it. Then installs ledgerblue in a virtualenv.
-            "command": "[ -n '$VIRTUAL_ENV' ] || if ! python3 -m virtualenv --version >/dev/null 2>&1; then python3 -m pip install virtualenv; fi && [ -d 'ledger' ] || python3 -m virtualenv ledger && source ledger/bin/activate && python3 -m pip show ledgerblue >/dev/null 2>&1 || python3 -m pip install ledgerblue",
+            // Installs functional tests python requirements in the docker container. 
+            "command":"docker exec -it ${config:container_name} bash -c 'export BOLOS_SDK=$(echo ${input:sdk}) && make -f.vscode/bolos_config.make configure'",
             "group": {
                 "kind": "test",
                 "isDefault": true
             },
             "problemMatcher": []
         },
         {
-            "label": "Install app loading requirements (Windows with PowerShell)",
+            "label": "[setup] Install tests requirements",
             "type": "shell",
-            // Checks that virtual env is installed, otherwise installs it. Then installs ledgerblue in a virtualenv.
-            "command": "cmd.exe /C 'if not exist ledger (python -m pip install virtualenv && python -m venv ledger && call ledger\\Scripts\\activate.bat && python -m pip install ledgerblue)'",
+            // Installs functional tests python requirements in the docker container. 
```

Comment:
This task should stay out of the helper tasks (and be in the `build` ones) so it is visible to users with the classic `ctrl + shift + b` shortcut.